pkgs/tools/backup/gphotos-sync/default.nix

```
@@ -0,0 +1,28 @@
+{ lib, python3Packages }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "gphotos-sync";
+  version = "2.14.2";
+
+  src = python3Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "1xq2mb6241f3gs7a7snqv177j4nqlvykxyhy5cw3d1krnzgp6dvi";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    appdirs
+    attrs
+    exif
+    psutil
+    pyyaml
+    requests_oauthlib
+  ];
+  pythonImportsCheck = [ "gphotos" ];
```

```suggestion
  ];

  pythonImportsCheck = [ "gphotos" ];
```

Are there tests we can run?

---

Yup, redone to use the full source and run tests.